scripts/configure_dev.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env bash
 set -e
 
+./scripts/check_golang_version.sh
```

Comment:
This is the human case where we should be flexible. I think it's normal for a new developer to run configure_dev.sh. The travis cases pinning to a specific version are just a check that travis is actually doing what we said to, right?

Reply:
Yes, that's a good point.

There should be at least a minimum version check.  For instance, if I try to install on ubuntu:18.04 using the version of go that I've downloaded from `apt`, I'm going to be working with go1.10 and that causes other failures when running `make`.

What do you and @tsachiherman feel would be a good minimum version?

Reply:
The minimal value would be one that is capable of building our source code. So it has to be above 0.13. I dont know if we require 0.14 or not.

Reply:
Going forward >= 0.14 would be good. golang.org is currently only releasing security patches to 0.14 and 0.15. I think this signals that everyone should migrate off 0.13

Reply:
I will make it so :)